### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by yashodhicy**
1. follow gitflow
2. add webpack 
3. add styles for the leaderboard app
4. implement add and rendering functions with dom creation
5.update Readme file

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: path.resolve(__dirname, 'src/images'),
```
Comment (topeogunleye) [inline webpack.config.js:18]: - [ ] [OPTIONAL] It seems that there is an issue with locating the 'images' folder in the 'src' directory when running the application. However, according to the webpack configuration file (`webpack.config.js`), the 'images' folder should be present in the 'src' directory.

To resolve this issue, please double-check the following:

1. Ensure that the 'images' folder exists in the 'src' directory.
2. Confirm that the 'images' folder contains the necessary image files.
3. Verify that the 'images' folder and its contents are correctly spelled and capitalized.

If the issue persists, please provide additional information or the relevant code snippet, so we can assist you further. Good luck!


image

**Review by topeogunleye — APPROVED** [review]
Hi @yashodhicy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that DOESN'T use microverse README file. Please kindly add the microverse README file,

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by topeogunleye** [comment]
Sorry for the initial approval @yashodhicy. I had to invalidate because you are not using the mIcoverse readme template

**Comment by yashodhicy** [comment]
> Sorry for the initial approval @yashodhicy. I had to invalidate because you are not using the mIcoverse readme template

@topeogunleye That's actually my mistake. I updated the readme but I forgot to push the changes to GitHub.

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @yashodhicy,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 

- Used webpack to bundle JavaScript. ✔️
- webpack, webpack-cli, webpack-dev-serve included in package.json. ✔️
- Has npm run scripts. ✔️
- node_modules folder is added to .gitignore ✔️
- Has basic project structure. ✔️
- All linters are passing ✔️
- Good commit history ✔️
- Followed Github flow ✔️
- Professional README file. ✔️
- PR has a good title ✔️

## [Optional] suggestions:

 * Nothing to mention.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**